src/app/item-triage/ValueDial.tsx

```diff
@@ -1,10 +1,11 @@
 import { AppIcon, banIcon, tagIcon } from 'app/shell/icons';
+import { clamp } from 'app/utils/util';
 import React from 'react';
 import styles from './ValueDial.m.scss';
 
 /** expects a value from 1-100 but caps both ends */
 export default function ValueDial({ value }: { value: number }) {
-  value = Math.min(100, Math.max(0, value));
+  value = clamp(value, 0, 100);
```

Comment:
@nev-r should this have been?
```suggestion
  value = clamp(value, 1, 100);
```
 based on the comment above